components/tools/OmeroPy/src/omero/plugins/ldap.py

```diff
@@ -102,6 +109,7 @@ def active(self, args):
         else:
             self.ctx.die(1, "No")
 
+    @windows_warning
```

Comment:
I'm not exactly sure of the distinction you mean, but in most cases, ldap.py is a consumer of the API as opposed to a local operation.